src/app/zap-templates/zcl/data-model/chip/matter-devices.xml

```diff
@@ -2209,6 +2209,12 @@ limitations under the License.
         <deviceId editable="false">0x0070</deviceId>
         <class>Simple</class>
         <scope>Endpoint</scope>
+        <endpointComposition>
+            <compositionType fullFamily="false", tree="true"/>
```

Comment:
This is not valid XML.

Also, seems like `<endpointComposition compositionType="type">` or `<compositionType value="type">` or `<compositionType>type</compositionType>` would all be better than having two booleans that can both be set to true (which would be very wrong).

Reply:
Okay thank you